src/fontra/client/core/server-utils.js

```diff
@@ -24,3 +24,7 @@ export const getCodePointFromGlyphName = memoize(async (glyphName) => {
 export async function parseClipboard(data) {
   return await callServerAPI("parseClipboard", { data });
 }
+
+export async function unionPath(data, contourIndices) {
```

Comment:
I think any path subsetting (which contours to even consider) should be done at the front end, and you should not pass contourIndices. Also, you should pass a _path_, not a _glyph_, just like I explicitly suggested in https://github.com/googlefonts/fontra/issues/1514#issuecomment-2258450907.

Reply:
Thanks. done.